site-src/concepts/conformance.md

```diff
@@ -0,0 +1,101 @@
+# Conformance
+
+This API covers a broad set of features and use cases and has been implemented
+widely. This combination of both a large feature set and variety of
+implementations requires clear conformance definitions and tests to ensure the
+API provides a consistent experience wherever it is used.
+
+When considering Gateway API conformance, there are three important concepts:
+
+## 1. Release Channels
+
+Within Gateway API, release channels are used to indicate the stability of a
+field or resource. The "standard" channel of the API includes fields and
+resources that have graduated to "beta". The "experimental" channel of the API
```

Comment:
link to k8s notion of beta stability

Reply:
I think the best thing to link to here would be https://kubernetes.io/docs/reference/using-api/#api-versioning, but it's not directly about beta stability. That is linked from our own versioning docs at the end of this paragraph, which I think may be sufficient here.